lms/envs/common.py

```diff
@@ -1144,6 +1141,10 @@
 
     # catches any uncaught RateLimitExceptions and returns a 403 instead of a 500
     'ratelimitbackend.middleware.RateLimitMiddleware',
+
+    # django current site middleware with default site
+    'django_sites_extensions.middleware.CurrentSiteWithDefaultMiddleware',
```

Comment:
As with the comment above, see edx/edx-django-sites-extensions#6 and edx/edx-platform#12555